lib/config.py

```diff
@@ -132,6 +144,26 @@ def parse_options(type):
     for key, credentials in _get_parameters_credentials().items():
         for name, description in credentials.items():
             parser.add_argument('--{}'.format(name), help=description)
+
+    return parser
+
+def parse_options(type):
```

Comment:
@jayeshjk : For blob operation, can we keep the cleanup part as it is now?

Reply:
Done.